### PR TITLE
Acquisitions tests update

### PR DESCRIPTION
**Pull request by Mullefa**
cc @guardian/contributions

## What does this change?

Extends the thank you Epic; removes the This is Your Land Epic.

## What is the value of this and can you measure success?

Continues to thank members; stops the This is Your Land campaign (respectively)

## Does this affect other platforms - Amp, Apps, etc?

No

## Screenshots

N/A

## Tested in CODE?

No

### CODE REVIEW

File: common/app/conf/switches/ABTestSwitches.scala [hunk 1]
```diff
@@ -145,7 +135,7 @@ trait ABTestSwitches {
     "Bootstrap the AB test framework to use the Epic to thank readers who have already supported the Guardian",
     owners = Seq(Owner.withGithub("Mullefa")),
     safeState = On,
-    sellByDate = new LocalDate(2017, 8, 1),
```
Comment (joelochlann) [inline common/app/conf/switches/ABTestSwitches.scala]: Could this go to a far-future date? Can't see us switching this off anytime soon

**Review by joelochlann — APPROVED** [review]
Fine to merge, I would consider putting the expiry dates in the far future though

File: common/app/conf/switches/ABTestSwitches.scala (hunk 1 above)
Comment (Mullefa) [inline common/app/conf/switches/ABTestSwitches.scala]: good point - will check with Amanda and extend if indefinitely if agreed

**Comment by Mullefa** [comment]
@guardian/guardian-frontend  Spoke to @SiAdcock and he said he'd be happier if I merged this tomorrow morning - it was 6ish when I spoke to him - sorry for leaving it so late. Feel free to merge if this becomes a blocker, otherwise I'll merge at about half 9.

**Pull request merged**